lua/ulx_yeet/sh_yeet.lua

```diff
@@ -2,7 +2,7 @@ if SERVER then
     ULib.ucl.registerAccess( "physgunragdollplayer", ULib.ACCESS_ADMIN, "Ability to physgun ragdoll other players.", "Other" )
 end
 
-local ragdollVelocity = CreateConVar( "ulx_physgun_ragdoll_velocity", 50, { FCVAR_REPLICATED, FCVAR_ARCHIVE }, "The velocity required for a physgunned player to turn into a ragdoll on release.", 0 ):GetInt()
+local ragdollVelocity = CreateConVar( "ulx_physgun_ragdoll_velocity", 40, { FCVAR_REPLICATED, FCVAR_ARCHIVE }, "The velocity required for a physgunned player to turn into a ragdoll on release.", 0 ):GetInt()
```

Comment:
Keeping `:GetInt()` here means we can't change this realtime.

Easiest way to handle this is to `:GetInt()` where we need to use it.
Middle-ground option is to make a function, `getRagdollVelocity` that does it for you.

Reply:
what? look line 7

Reply:
Sure I suppose that works. You entirely lose the reference to this convar, but if you never need it again I guess it's okay